### ISSUE TITLE
write seems to fail with Decimal value

### ISSUE THREAD

**Issue by ghost** [influxdata/influxdb-python#482]
Hi,

As reported by @chripede in #430 , using Decimal value with write triggers an exception:
e.g if the value is 2695.30
```
influxdb.exceptions.InfluxDBClientError: 400: {"error":"partial write: unable to parse 'totalgain value=Decimal('2695.30')': invalid boolean dropped=0"}
```

See the last message of @chripede in #430 for another example. As he said:

> I think the problem might be that float() will return a float on a Decimal, but repr will actually return "Decimal('29.95')"

Let me know if you need more information.

Best Regards,
Carl

**Comment by sebito91** [comment]
Commented on your #488 PR, why not just return the `repr(float(value))` since you've already passed the check?

**Comment by sebito91** [comment]
Fixed in #813
